CHANGELOG.md

```diff
@@ -16,8 +16,7 @@ Now:
 
 ```js
 gheets.getWorksheet('ABC', 'xyz')
-  .then(result => {})
-  .catch(error => {});
+  .then(result => {}, error => {});
 ```
```

Comment:
I disagree. Depending on what you do in your success callback, you may not want to catch that error ([case in point](https://twitter.com/dan_abramov/status/798839704326049794)! That's why I updated the example to only handle errors from gsheets itself.